providers/implementations/macs/kmac_prov.c

```diff
@@ -244,8 +257,15 @@ static int kmac_setkey(struct kmac_data_st *kctx, const unsigned char *key,
 {
     const EVP_MD *digest = ossl_prov_digest_md(&kctx->digest);
     int w = EVP_MD_get_block_size(digest);
-
-    if (keylen < KMAC_MIN_KEY || keylen > KMAC_MAX_KEY) {
+    const size_t minkeylen =
+#ifdef FIPS_PROVIDER
```

Comment:
FIPS_PROVIDER seems like the wrong define?

Reply:
Oops.  Fixed.